src/cript/api/api.py

```diff
@@ -31,7 +34,12 @@
 )
 from cript.api.utils.web_file_downloader import download_file_from_url
 from cript.api.valid_search_modes import SearchModes
+from cript.nodes.primary_nodes.primary_base_node import PrimaryBaseNode
+
 from cript.nodes.primary_nodes.project import Project
+from deepdiff import DeepDiff
+import cript
```

Comment:
you can't import cript here.
That's a circular import.

Be specific and import exactly what you need.